cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -235,6 +235,11 @@ spec:
         # Enable deployment of snapshotter container in ceph-csi provisioner
         - name: CSI_ENABLE_SNAPSHOTTER
           value: "true"
+        # Enable Ceph Kernel clients on kernel < 4.17 which support quotas for Cephfs
```

Comment:
Could we have the kernel version requirement for this default feature added to the documentation "ceph prequisites" page?